release-process.rst

```diff
@@ -5,11 +5,17 @@
 .. contents::
    :depth: 2
 
-:From:
-   https://wiki.php.net/rfc/releaseprocess
-
-:Updated by:
-   https://wiki.php.net/rfc/release_cycle_update
+Sources
+
++--------------+---------------------------------------------------------------+
+| Original RFC | -  `Request for Comments: Release Process                     |
+|              |    <https://wiki.php.net/rfc/releaseprocess>`_                |
++--------------+---------------------------------------------------------------+
+| Updates      | -  `RFC: Release Cycle Update                                 |
+|              |    <https://wiki.php.net/rfc/release_cycle_update>`_          |
+|              | -  `RFC: Policy Release Process Update                        |
+|              |    <https://wiki.php.net/rfc/policy-release-process-update>`_ |
++--------------+---------------------------------------------------------------+
```

Comment:
Can we not use `list-table`? changes to formatted tables become very noisy.

Reply:
I would just have done this:

```
*Original RFC:* `Request for Comments: Release Process <https://wiki.php.net/rfc/releaseprocess>`_

*Updated By:*

- `RFC: Release Cycle Update <https://wiki.php.net/rfc/release_cycle_update>`_
- `RFC: Policy Release Process Update <https://wiki.php.net/rfc/policy-release-process-update>`_
```

It doesn't have to be complicated.

Reply:
I tried and it looks ugly.

Reply:
See https://github.com/php/policies/blob/5a4bdb7a5d518665bf806cdf3f7455025d892ea4/release-process.rst . Basically it looks like it's part of the content.